itou/api/employee_record_api/tests/tests.py

```diff
@@ -246,6 +246,23 @@ def test_show_phone_email_api(self, _mock):
         self.assertEqual(results.get("adresse").get("adrTelephone"), self.user.phone)
         self.assertEqual(results.get("adresse").get("adrMail"), self.user.email)
 
+    @mock.patch(
+        "itou.common_apps.address.format.get_geocoding_data",
+        side_effect=mock_get_geocoding_data,
+    )
+    def test_missing_approval(self, _mock):
+        job_application = JobApplicationWithCompleteJobSeekerProfileFactory(to_siae=self.siae)
+        employee_record_sent = EmployeeRecord.from_job_application(job_application=job_application)
+        employee_record_sent.update_as_ready()
+
+        job_application.approval.delete()
+
+        self.client.login(username=self.siae_member.username, password=DEFAULT_PASSWORD)
+        response = self.client.get(ENDPOINT_URL + "?status=READY", format="json")
+        self.assertEqual(response.status_code, 200)
+        json = response.json()
+        self.assertEqual(len(json.get("results")), 2)
```

Comment:
j'ai un peu du mal à comprendre ce commit !
en gros on vérifie quoi ici, que l'on ne se prend pas d'erreur 500 c'est ça ? C'est ce qui arrivait avant ?
parce que du coup je serais content que l'on vérifie quelque chose sur les objets sérialisés, par exemple que l'on peut avoir dans le JSON un `passDateFin` à None. Est-ce le cas ?

Peut-etre que ce qui m'induit en erreur est le titre de commit qui parle de dates optionnelles alors que ici tu supprimes carrément le PASS ? CF le nom du test ^^ 

Bref, peut etre que un peu plus de précision ou un commentaire me permettrait de comprendre le lien entre les deux. 

N'hésite pas à dire de quelle section on parle dans le titre du commit, comme `employee_record_api: make approval dates optional` pour que l'on sache (notamment au moment du changelog) de quoi on parle. On n'a pas rendu toutes les dates des PASS IAE optionnelles, ce que le titre du commit semble dire !

Reply:
Merci Victor pour tes questions (que j'espérais en secret :D) 
Ce [test qui casse la CI](https://github.com/betagouv/itou/blob/4d54ca11e4fb25f1c277db719259b2edd28b7a47/itou/employee_record/tests/tests_update_management_command.py#L119) me fait penser que c'est pas une excellente idée le `required=False`. 
Post-it : prévoir un point avec Fred :-|